lib/opensusebasetest.pm

```diff
@@ -1292,7 +1292,7 @@ use root-ssh console directly.
 =cut
 
 sub select_serial_terminal {
-    my ($self, $root) = @_;
+    my ($root) = @_;
```

Comment:
Maybe we should allow the call as static function and as member method?

```suggestion
    my ($self, $root) = @_
    $root = $self if (ref($root) ne 'opensusebasetest');
```

Reply:
This would solve the issue I have in [#15684](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/15684#discussion_r996648918).

Reply:
Thank you Clemens for interesting solution but as you also [suggested](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/15725#issuecomment-1283585825) I proceed otherwise.